daffodil-core/src/main/scala/org/apache/daffodil/dpath/DFDLExpressionParser.scala

```diff
@@ -235,6 +242,7 @@ class DFDLPathExpressionParser[T <: AnyRef](qn: NamedQName,
     })("add")
 
   def MultiplicativeExpr: Parser[Expression] = log(
+    UnaryExpr ~ (("/") ~ UnaryExpr) ^^ { _ => context.SDE("'/' is not a valid operator in DFDL Expression Syntax. Did you mean to use 'div'?") } |
```

Comment:
Like Steve suggested, this seems like the more correct way to handle this, but it causes all the tests involving paths to break.

Reply:
Yeah, I was afraid that would happen. I'm not sure this is really worth fixing. The ambiguity with path expressions makes this difficult to resolve. I believe we have a general bug to fix diagnostics coming out of DFDL expression compilation. I suspect whatever is done to make that better will be the right solution to also make this specific case better.

Reply:
That's kind of what I was leaning towards too, but figured I would see if there was a reasonably simple way to add this kind of helpful error message.  I'll see if I can find the bug for better diagnostics and I'll mark the ticket for / vs div as a duplicate or something.